RallyHereIntegration/Source/RallyHereIntegration/Private/RH_GameInstanceBootstrappers.cpp

```diff
@@ -136,13 +141,24 @@ void URH_GameInstanceServerBootstrapper::Initialize()
 			, *RH_GETENUMSTRING("/Script/RallyHereIntegration", "ERH_ServerBootstrapMode", BootstrapMode));
 	}
 
-	if (FParse::Value(FCommandLine::Get(), TEXT("rhmaxrecyclecount="), MaxRecycleCount))
+	if (FParse::Value(FCommandLine::Get(), TEXT("rh.maxrecyclecount="), MaxRecycleCount))
 	{
 		UE_LOG(LogRallyHereIntegration, Log, TEXT("[%s] - Max recycle count overridden by commandline to %d"), ANSI_TO_TCHAR(__FUNCTION__), MaxRecycleCount);
 	}
 
 	SetTerminationSignalHandler();
 
+#if UE_TRACE_ENABLED
+	FTraceAuxiliary::OnTraceStopped.AddWeakLambda(this, [this](FTraceAuxiliary::EConnectionType TraceType, const FString& TraceDestination)
+		{
+			if (TraceType == FTraceAuxiliary::EConnectionType::File)
+			{
+				// upload the trace file
+				ConditionalAutoUploadTraceFile(TraceDestination);
```

Comment:
Will our objects still be around when this is called?

Reply:
They likely will (unless trace file is being closed on shutdown), but this is a weak lambda so it will only get called if the object is still valid.